spec/Functions.tex

```diff
@@ -734,13 +734,6 @@ \subsection{The Ref Return Intent}
 behavior can be useful for adjusting the behavior depending on the
 calling context.
 
-\begin{craychapel}
-Previous versions of Chapel included an implicit \chpl{setter} param of
-type \chpl{bool} for \chpl{ref} return intent functions. The old behavior
-can be recreated by making both ref-pair functions call another function
-and explicitly pass the \chpl{setter} value, as in the the example below.
-\end{craychapel}
-
```

Comment:
There's a reference to a setter argument further on in this file (line 772 ish).  Is that section still interesting?